heatmaps-troubleshooting-section (#16762)
DataDog/documentation: content/en/real_user_monitoring/heatmaps/_index.md

## Troubleshooting

### I am looking at a heatmap for a given view, but it’s showing me an unexpected page.

Heatmaps are based on RUM view names and depending on how your RUM Application is configured, many pages can start being grouped under the same view name, or you can start having very specific view names. If you think the default view name gathering is not sufficient, you can override it manually with the [startView][6] function. 

### After attempting to create a heatmap, I see a "No Replay Data" state appear. 

This means that we could not find any Session Replay to use as a Heatmap background that matches the current search filters. If you just started to record sessions with the [Browser SDK][2], it may also take a few minutes for the Session Replay to be available for viewing.

### After attempting to create a heatmap, I see a "Not enough data to generate a heatmap" state appear.

This means that we weren't able to match any user actions with the current selected replay. It happens for a variety of reasons such as:

> - Your application may not be using the latest SDK version (>= 4.20.0)

Reviewer: ```suggestion
- Your application is not using the latest SDK version (>= 4.20.0).
```

> - RUM actions are not enabled (learn how to [track user interactions][7])

Reviewer: ```suggestion
- RUM actions are not enabled. Learn how to [track user interactions][7].
```

> - Your page has changed drastically recently 

Reviewer: ```suggestion
- Your page has recently changed drastically. 
```

### All of the user information on the page is empty.

> User information is not collected by default. Heatmaps leverage the user information available in your session data to display relevant insights on behavior. You can set up user information in RUM by following [these steps][8].

Reviewer: ```suggestion
User information is not collected by default. Heatmaps use the user information available in your session data to display relevant insights on behavior. You can set up user information in RUM by following the steps in [Modifying Data and Context][8].
```
